grabber/Gopkg.lock

```diff
@@ -20,7 +20,7 @@
   version = "v1.7.0"
 
 [[projects]]
-  branch = "optimize-holders"
+  branch = "extend-api"
   name = "github.com/gochain-io/explorer"
   packages = [
     "server/backend",
```

Comment:
It might be cleaner and more idiomatic to run `dep` from the top level in one place for everything - then these inter-repo reference go away.